0-common/igluctl/src/main/scala/com.snowplowanalytics.iglu/ctl/PushCommand.scala

```diff
@@ -114,7 +114,7 @@ case class PushCommand(registryRoot: HttpUrl, masterApiKey: UUID, inputDir: File
   def buildRequest(schema: IgluSchema, writeKey: String): HttpRequest @@ PostSchema = {
     val request = Http(s"$registryRoot/api/schemas/${schema.self.toPath}")
       .header("apikey", writeKey)
-      .param("isPublic", isPublic.toString)
+      .param("is_public", isPublic.toString)
```

Comment:
It [exists](https://github.com/snowplow/iglu/blob/release/r9/0-common/igluctl/src/main/scala/com.snowplowanalytics.iglu/ctl/PushCommand.scala#L117) @chuwy , why shouldn't it be? This change isn't user facing. I guess you refer to my [initial](https://github.com/snowplow/iglu/blob/2b1c700e49e5c5480308e6dfef0f01fc3ff04969/0-common/igluctl/src/main/scala/com.snowplowanalytics.iglu/ctl/PushCommand.scala#L84) work where I converted `vendor_prefix` to `vendorPrefix`.

Reply:
Sorry, not following. How is it not user-facing change when Igluctl uses camel-case syntax? https://github.com/snowplow/iglu/blob/master/0-common/igluctl/src/main/scala/com.snowplowanalytics.iglu/ctl/PushCommand.scala#L117

Reply:
Sorry. Deleted the comment. What Scala Registry uses? If any version of these two apps use different cases - it is a user-facing change (and API-breaking).

Reply:
You're right, I thought of CLI for a moment. It doesn't matter (in terms of compatibility) which parameter we changed, making parameters consistent will result in igluctl sending request with either of parameters changed, an API breaking change. Should we also keep previous parameter name to be compatible with previous igluctl versions?

Reply:
This is very important: all existing (0.1.0-0.4.0) versions of Igluctl must be compatible with upcoming Iglu Registry 0.3.0. Also all existing versions of Iglu Registry must be compatible with upcoming Igluctl (0.4.1), absolutely no breaking changes.

Whenever we'll decide to break compatibility we should have very strong reasons to do so and do it just once on all fronts.

Reply:
Considering that making query parameters isn't a strong reason, I'd keep `isPublic` (old) beside `is_public` (new) so that registry will stay compatible with previous versions and Swagger UI will use old name to stay `UI compatible` (if there is any phrase like this).

In this case, do we really achieve to make query parameters consistent or do we make them a mess? I think it is latter and I propose to abandon this change. What do you think @chuwy ?

Reply:
Cannot we change `vendor_prefix` instead? IIRC it wasn't used anywhere.

Reply:
`vendor_prefix` is being used to create temporary read/write keys.

Reply:
Oh, makes sense then. Then yes, let's push back this change. Also please assign "breaking-change" label to it, I'd like to handle all API breaking issues at once in future.

Reply:
Gracefully pushed back.

Reply:
Thanks!